package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "urbit-key-generation",
-  "version": "0.11.1",
+  "version": "0.11.2",
```

Comment:
Might want to bump to `0.12` for this technically breaking change.

Reply:
On second thought: because we're fixing a security vulnerability, I think this might indeed count as "substantial."  Minor bump it is!